app/controllers/OrganizationController.scala

```diff
@@ -64,6 +65,23 @@ class OrganizationController @Inject()(organizationDAO: OrganizationDAO,
     } yield Ok(Json.toJson(js))
   }
 
+  case class OrganizationCreationParameters(organization: Option[String],
+                                            organizationDisplayName: String,
+                                            ownerEmail: String)
+  object OrganizationCreationParameters {
+    implicit val jsonFormat: OFormat[OrganizationCreationParameters] = Json.format[OrganizationCreationParameters]
+  }
+  def create: Action[OrganizationCreationParameters] =
+    sil.SecuredAction.async(validateJson[OrganizationCreationParameters]) { implicit request =>
+      for {
+        _ <- userService.assertIsSuperUser(request.identity._multiUser) ?~> "notAllowed" ~> FORBIDDEN
+        owner <- multiUserDAO.findOneByEmail(request.body.ownerEmail) ?~> "user.notFound"
+        org <- organizationService.createOrganization(request.body.organization, request.body.organizationDisplayName)
+        user <- userDAO.findFirstByMultiUser(owner._id)
+        _ <- userService.joinOrganization(user, org._id, autoActivate = true, isAdmin = true)
```

Comment:
```suggestion
        _ <- userService.joinOrganization(user, org._id, autoActivate = true, isAdmin = true, isOwner = true)
```
(should be propagated to the user’s isOrganizationOwner field)